### PR TITLE
Os_tips: Change the type of 'onclose'

### PR DESCRIPTION
**Pull request by Zzull**
I need to animate with `Lwt` the closing of a tip

### CODE REVIEW

File: opam [hunk 1]
```diff
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "ocsigen-start"
-version: "3.0.0"
```
Comment (jrochel) [inline opam:3]: Pour un breaking change il suffit d'augmenter la version mineur : 3.1.0

**Review by jrochel — CHANGES_REQUESTED** [review]

File: opam (hunk 1 above)
Comment (jrochel) [reply opam:3]: (Si je ne me trompe pas. C'est peut-être différent pour Ocsigen.)

Comment (Zzull) [reply opam:3]: What warrants a major version then?

Comment (jrochel) [reply opam:3]: Some sort of paradigm shift for example? Again, I vaguely remember @balat saying that for Ocsigen every breaking change requires an increase of the major version.

Comment (vouillon) [reply opam:3]: https://semver.org/
> Given a version number MAJOR.MINOR.PATCH, increment the:
>
>    MAJOR version when you make incompatible API changes,
>    MINOR version when you add functionality in a backwards compatible manner, and
>    PATCH version when you make backwards compatible bug fixes.

**Pull request merged**